rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -167,23 +170,26 @@ def _recursively_initialize_decoders(
                     address_results.update(self.decoders[class_name].addresses_to_decoders())
                     rules_results.extend(self.decoders[class_name].decoding_rules())
                     enricher_results.extend(self.decoders[class_name].enricher_rules())
+                    post_decoding_results.extend(self.decoders[class_name].post_decoding_rules())  # noqa: E501
                     self.all_counterparties.update(self.decoders[class_name].counterparties())
 
-                recursive_addrs, recursive_rules, recurisve_enricher_results = self._recursively_initialize_decoders(full_name)  # noqa: E501
+                recursive_addrs, recursive_rules, recurisve_enricher_results, recursive_post_decoding_results = self._recursively_initialize_decoders(full_name)  # noqa: E501
```

Comment:
I know it was from before but please fix this typo
```suggestion
                recursive_addrs, recursive_rules, recursive_enricher_results, recursive_post_decoding_results = self._recursively_initialize_decoders(full_name)  # noqa: E501  
```

Also perhaps it makes sense to change this and add all those into a dataclass or named tuple. It's kind of silly to return a raw tuple of so many things and pass it around.